test/DynamoCoreWpfTests/DynamoCoreWpfTests.csproj

```diff
@@ -37,25 +40,9 @@
 	  <PackageReference Include="SharpDX.Direct3D9" Version="4.2.0" />
 	  <PackageReference Include="SharpDX.DXGI" Version="4.2.0" />
 	  <PackageReference Include="SharpDX.Mathematics" Version="4.2.0" />
-      <PackageReference Include="NUnitTestAdapter" Version="2.3.0" ExcludeAssets="all" />
+      <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" IncludeAssets="none" />
```

Comment:
not sure you need to set `IncludeAssets = none` - just looking at the other references to this.